docs/docs/community/create-provider.md

```diff
@@ -161,3 +164,82 @@ export class TwilioSmsProvider implements ISmsProvider {
 ```
 
 [GitHub Template](https://github.com/novuhq/provider-template)
+
+## Add provider logos
+In order to present the provider in the Integration store we need logos in dark and light mode, 
+this step is desirable but not necessary you can add and we will check and update if necessary. 
+If you locate the logos you can add them to `apps/web/public/static/images/providers` while the name of the file
+is the name of the provider.
+The possible formats are svg and png.
+
+
+## Add config item with in the list
+In order to build the UI integration store we need to provide it with list of provider integration.
+This part is made up of two parts and the first is to create credentials config and the second one is to add
+provider configuration to providers list.
+
+### Create credentials config
+We need to add the credentials that are needed in order to create integration with the provider. For example if you 
+added email provider like SendGrid and the credentials are 'From', 'SenderName' and 'ApiKey' you will need to add 
+a config object in `libs/shared/src/consts/providers/provider-credentials.ts` like below.
+
+```typescript
+export const sendgridConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.ApiKey,
+    displayName: 'API Key',
+    type: 'string',
+  },
+  ...mailConfigBase,
+];
+```
+
+
+### Add provider to providers list
+Now we need to add the provider data into the list that located in `libs/shared/src/consts/providers/channels/email.ts`.
+Note that the id is the providers name, displayName is providers name in pascal case, credentials are the one 
+you created on the previous step, logoFileName should be as it was on the adding logo step 
+(with the format type included).
+
+```typescript
+ {
+    id: 'sendgrid',
+    displayName: 'SendGrid',
+    channel: ChannelTypeEnum.EMAIL,
+    credentials: sendgridConfig,
+    docReference: 'https://docs.sendgrid.com/',
+    logoFileName: { light: 'sendgrid.png', dark: 'sendgrid.png' }
+}
+```
+
+
+## Add provider handler in the API
+
+### Create provider handler
```

Comment:
@djabarovgeorge could you add a step explaining how to add a provider dependency to the API?
Adding the dependency manually to the package.json with the version specified in the provider package, and the running npm run setup:project from the root folder. 

People had a lot of trouble with this.

Reply:
Sure sound like a great idea!